src-ThirdParty/StrongNameSigner/SigningHelper.cs

```diff
@@ -0,0 +1,33 @@
+using System;
```

Comment:
I think it would be useful to have comments linking to the original version of the source file, e.g. https://github.com/brutaldev/StrongNameSigner/blob/c38d42ab8d1444504720a62736b310303236cd85/src/Brutal.Dev.StrongNameSigner/SigningHelper.cs#L437 ?